runtime/client/src/lttexturestring.h

```diff
@@ -12,6 +12,12 @@
 #	include "ilttexturestring.h"
 #endif
 
+#ifdef __LINUX
+#define LT_TEXTURE_STRING_TYPE char
+#else
+#define LT_TEXTURE_STRING_TYPE wchar_t
```

Comment:
better yet use a typedef
that way the IDE has an easier time helping you with type information.
```cpp
#ifndef __LINUX
typedef LTTextureStrType wchar_t;
#else
typedef LTTextureStrType char;
#endif
```